docs/userguide/simulation.md

```diff
@@ -13,36 +13,99 @@ Thus, "simulation" means investigating what experimental trajectory we would hav
 
 2. It can refer to the simulation of an *actual* DOE loop, i.e., recommending experiments and retrieving the corresponding measurements, where the loop closure is realized in the form of a callable (black-box) function that can be queried during the optimization to provide target values. Such a callable could for instance be a simple analytical function or a numerical solver of a set of differential equations that describe a physical system.
 
-## The Lookup Functionality
+## The Lookup Mechanism
 
-In BayBE, the simulation submodule allows a wide range of use cases and can even be used for "oracle predictions".
-This is enabled by the proper use of the `lookup` functionality, which allows to either use fixed data sets, analytical functions, and general callbacks for retrieving target function values. 
+BayBE's simulation package enables a wide range of use cases and can even be used for "oracle predictions".
+This is made possible through the flexible use of `lookup` mechanisms, which act as the loop-closing element of an optimization loop.
 
-All functions require a `lookup` which is used to close the loop and return target values for points in the search space.
-It can be provided in the form of a dataframe or a `Callable`.
+Lookups can be provided in a variety of ways, by using fixed data sets, analytical functions, or any other form of black-box callable.
+In all cases, their role is the same: to retrieve target values for parameter configurations suggested by the recommendation engine.
 
-```{note}
-Technically, the `lookup` can also be `None`. This results in the simulation producing random results which is not discussed further.
+### Using a `Callable`
+
+Using a `Callable` is the most general way to provide a lookup mechanism.
+Any callable is a suitable lookup as long as it accepts a dataframe containing parameter configurations and returns the corresponding target values.
```

Comment:
Have changed it though I don't consider it strictly necessary – basically it can be `Callable` to refer to the actual python object and "callable" to refer to the concept. Just like "discrete numerical parameter" and `DiscreteNumericalParameter`. In fact, there are other places in the same file where it is still "callable". But now at least the two occurrences close next to each other are synced.